### ISSUE TITLE
Bug: self-assignment of method parameter

### ISSUE THREAD

**Issue by mstr2** [javafxports/openjdk-jfx#422]
https://github.com/javafxports/openjdk-jfx/blob/4e6a8f319256b0eeb72a70f09b7e63680ba9a241/modules/javafx.graphics/src/main/java/javafx/scene/shape/Shape.java#L406

**Comment by nlisker** [comment]
Yes, this is a bug, but it has no effect since the method is called only by the shape's accessor, and its method isn't being called at all.
